src/Tribe/Commerce/PayPal/Errors.php

```diff
@@ -22,16 +22,16 @@ public static function error_code_to_message( $error_code = '-1' ) {
 		$map = array(
 			'-1'  => __( 'There was an error', 'event-tickets' ),
 			'1'   => __( 'Attendee email and/or full name is missing', 'event-tickets' ),
-			'2'   => __( 'Trying to oversell a ticket but the current oversell policy does not allow it', 'event-tickets' ),
-			'3'   => __( 'Ticket quantity is 0', 'event-tickets' ),
+			'2'   => sprintf( __( 'Trying to oversell a %s but the current oversell policy does not allow it', 'event-tickets' ), tribe_get_ticket_label_singular_lowercase( 'paypal_error_oversell_policy' ) ),
+			'3'   => sprintf( __( '%s quantity is 0', 'event-tickets' ), tribe_get_ticket_label_singular( 'paypal_error_zero_quantity' ) ),
```

Comment:
This only escapes the __ string, not the strings you are inserting via printf(). Can you follow the other examples where I've made suggestions that does it like `esc_html( sprintf( .... ) )` here elsewhere in your PR?